goodai/ltm/keywords/keywords.py

```diff
@@ -0,0 +1,75 @@
+import spacy
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+# Load the English model
+nlp = spacy.load("en_core_web_sm")
```

Comment:
We should consider the multi-language setting.

Reply:
Yeah that is something we will have to consider, I will create a ticket for it.

Reply:
I get an error while loading this. Did you have to install something?
```text
>>> nlp = spacy.load("en_core_web_sm")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dcastillo/miniconda3/envs/ltm/lib/python3.12/site-packages/spacy/__init__.py", line 51, in load
    return util.load_model(
           ^^^^^^^^^^^^^^^^
  File "/home/dcastillo/miniconda3/envs/ltm/lib/python3.12/site-packages/spacy/util.py", line 472, in load_model
    raise IOError(Errors.E050.format(name=name))
OSError: [E050] Can't find model 'en_core_web_sm'. It doesn't seem to be a Python package or a valid path to a data directory.
```

Reply:
Ah, nevermind...
`python -m spacy download en_core_web_sm`

Reply:
You can do `spacy.cli.download("en_core_web_sm")`, but then you need to restart the session, so it's advised to do it [through pip install](https://spacy.io/usage/models#download-pip). Maybe that can be added to `requirements.txt`.